lib/ProductOpener/Products.pm

```diff
@@ -2414,6 +2419,23 @@ sub index_product($product_ref)
 	return;
 }
 
+sub index_search_service($product_ref) {
```

Comment:
The name is not very explicit to me.

I would name it `push_to_search_service` or something like that.